packages/node-core/src/indexer/entities/Poi.entity.ts

```diff
@@ -10,16 +10,16 @@ export interface ProofOfIndex {
   parentHash?: Uint8Array;
   operationHashRoot: Uint8Array;
   mmrRoot?: Uint8Array;
-  projectId: string;
+  projectId?: string;
 }
 
 export interface PoiModel extends Model<ProofOfIndex>, ProofOfIndex {}
 
 export type PoiRepo = typeof Model & {
   new (values?: unknown, options?: BuildOptions): PoiModel;
 };
-
-export function PoiFactory(sequelize: Sequelize, schema: string): PoiRepo {
+// Project Id is deprecated
+export function PoiFactoryDeprecate(sequelize: Sequelize, schema: string): PoiRepo {
```

Comment:
I don't think we should keep a deprecated version. We should drop the column instead